services/ui-src/src/components/alerts/Alert.tsx

```diff
@@ -94,6 +96,7 @@ const sx = {
     position: "absolute",
     color: "palette.base",
     marginBottom: "1.75rem",
+    width: "1.2rem",
```

Comment:
this'll give us a partial pixel, if that matters (19.2px)

Reply:
For the most part we've been trying to stick to rem values on the 4ths (.25, .5, .75) and _sometimes_ the 8ths when needed (.125, .375, .625, .875).